src/bflib_video.c

```diff
@@ -410,10 +402,10 @@ static LPCTSTR MsResourceMapping(int index)
 static TbResult LbScreenActivationUpdate(void)
 {
     SDL_Event ev;
-    ev.type = SDL_ACTIVEEVENT;
-    ev.active.state = SDL_APPACTIVE;
-    ev.active.gain = ((SDL_GetAppState() & ev.active.state) != 0);
-    SDL_PushEvent(&ev);
+    // ev.type = SDL_ACTIVEEVENT;
+    // ev.active.state = SDL_APPACTIVE;
```

Comment:
Maybe you just remove this lines? But please write a comment like "this function should activate game window but it is not needed anymore"

Reply:
Yep, I prefer deleting obsolete code instead of commenting it, dunno why I left it this time. Will remove it.

Reply:
ok